synchro/synchrotest.py

```diff
@@ -184,6 +184,8 @@
     "TestUnifiedFindShutdownError.test_Concurrent_shutdown_error_on_find",
     "TestUnifiedInsertShutdownError.test_Concurrent_shutdown_error_on_insert",
     "TestUnifiedPoolClearedError.test_PoolClearedError_does_not_mark_server_unknown",
+    # These tests require synchronous method calls.
```

Comment:
"This test requires a synchronous method call"

Reply:
Done.